config/webpack/webpack.dev.js

```diff
@@ -0,0 +1,14 @@
+const webpack = require('webpack');
+const merge = require('webpack-merge');
+const base = require('./webpack.base');
+
+module.exports = merge(base, {
+  mode: 'development',
+  devtool: 'eval-source-map',
+  plugins: [new webpack.HotModuleReplacementPlugin()],
+  devServer: {
+    contentBase: './dist',
+    hot: true,
+    open: true
+  }
```

Comment:
port: 8000, //can define port or it will falback to default
    overlay: true, //if we got error, it will show on screen
    historyApiFallback: true //need this for react router

Reply:
Thanks. Updated.